util/reflect.go

```diff
@@ -458,9 +458,24 @@ func DeepEqualDerefPtrs(a, b interface{}) bool {
 // if the struct tag is invalid, or nil if tag is valid but the schema is not
 // found in the tree at the specified path.
 func ChildSchema(schema *yang.Entry, f reflect.StructField) (*yang.Entry, error) {
+	return ChildSchema2(schema, f, false)
+}
+
+// ChildSchema2 returns the schema for the struct field f, if f contains a valid
```

Comment:
Same issue with this function naming as above.

Reply:
Done, named it `ChildSchemaPreferShadow`